codegens/csharp-restsharp/test/unit/convert.test.js

```diff
@@ -3,6 +3,7 @@ var expect = require('chai').expect,
   convert = require('../../lib/index').convert,
   mainCollection = require('./fixtures/testcollection/collection.json'),
   testCollection = require('./fixtures/testcollection/collectionForEdge.json'),
+  uaTest = require('./fixtures/testUA.json'),
```

Comment:
I don't think we need a whole new file for this. You can add a const object in the test itself.